docs/data/material/components/buttons/buttons.md

```diff
@@ -10,141 +10,147 @@ unstyled: /base-ui/react-button/
+### Variants
 
-Outlined buttons are also a lower emphasis alternative to contained buttons,
-or a higher emphasis alternative to text buttons.
+The Button component supports three values for the `variant` prop: `text` (default), `contained`, and `outlined`.
 
-{{"demo": "OutlinedButtons.js"}}
+{{"demo": "BasicButtons.js"}}
 
-## Handling clicks
+#### Elevation
 
-All components accept an `onClick` handler that is applied to the root DOM element.
+Apply the `disableElevation` prop to remove the default drop shadow from the `contained` variant.
 
-```jsx
-<Button
-  onClick={() => {
-    alert('clicked');
-  }}
->
-  Click me
-</Button>
-```
+{{"demo": "DisableElevation.js"}}
 
-Note that the documentation [avoids](/material-ui/guides/api/#native-properties) mentioning native props (there are a lot) in the API section of the components.
+### Colors
 
-## Color
+Every palette included in the theme is available via the `color` prop.
 
 {{"demo": "ColorButtons.js"}}
 
-In addition to using the default button colors, you can add custom ones, or disable any you don't need. See the [Adding new colors](/material-ui/customization/palette/#custom-colors) examples for more info.
+:::success
+If the default colors aren't sufficient, you can add custom colors and remove defaults that you don't need.
+See the [Palette—Custom colors](/material-ui/customization/palette/#custom-colors) doc for more info.
+:::
 
-## Sizes
+### Sizes
 
-For larger or smaller buttons, use the `size` prop.
+The Button component comes in three sizes: `small`, `medium` (default), and `large`.
 
 {{"demo": "ButtonSizes.js"}}
 
-## Buttons with icons and label
+#### Width
 
-Sometimes you might want to have icons for certain buttons to enhance the UX of the application as we recognize logos more easily than plain text. For example, if you have a delete button you can label it with a dustbin icon.
+Use the `fullWidth` prop to stretch the Button to fill its container.
+
+{{"demo": "FullWidthButton.js"}}
+
+### Start and end icons
+
+Use the `startIcon` or `endIcon` prop to append an icon to the beginning or end of a Button's contents.
 
 {{"demo": "IconLabelButtons.js"}}
 
-## Icon button
+### Ripple effect
 
-Icon buttons are commonly found in app bars and toolbars.
+The ripple effect provides visual feedback to the user that the Button is being clicked or focused.
+Use the `disableFocusRipple` prop to remove only the focus effect, or `disableRipple` to remove the ripple when focusing and clicking.
 
-Icons are also appropriate for toggle buttons that allow a single choice to be selected or
-deselected, such as adding or removing a star to an item.
+:::warning
+These props remove _all_ default styles from the `:focus-visible` state, which means the user won't have any visual indication that the Button is focused.
+You can target the `.Mui-focusVisible` class to define your own styles for a better user experience.
```

Comment:
```suggestion
You can target the `.Mui-focusVisible` class to define your own styles for an accessible user experience.
```

Thought of calling out the accessibility impact of having properly styled focus styles here instead of saying "better"; maybe that's more specific... and better? 😅